ext/taglib_wav/taglib_wav.i

```diff
@@ -4,7 +4,7 @@
 #include <taglib/wavfile.h>
 #include <taglib/wavproperties.h>
 #include <taglib/id3v2tag.h>
-using namespace TagLib::RIFF;
+using StripTags = TagLib::File::StripTags;
```

Comment:
I think we're doing something wrong here. Swig is not "seeing" TagLib::File so it doesn't know what StripTags is. The proper solution would be to `%include <taglib/rifffile.h>` but then we run afoul of namespace flattening.

Maybe we need a separate .i for riff?

Since the test pass now, maybe we should fix this in a later PR.

Reply:
I figured out how to do it, it was a matter of `%ignore TagLib::RIFF::File`.